src/DssSpell.t.base.sol

```diff
@@ -1328,6 +1329,20 @@ contract DssSpellTestBase is Config, DssTest {
         assertEq(vestDai.rxd(_index), _claimed,           "rxd");
     }
 
+    function _checkTransferrableVestMkrAllowance() internal {
+        uint256 vestableAmt;
```

Comment:
```suggestion
        uint256 vestableAmt = 0;
```
Readability. Up to you

Reply:
yeah, the default value here should be zero, not to opposed for readability but I think we do the same in other codebase so maybe we could keep it as it is.

Reply:
Ok we can resolve this then